src/core_ocean/shared/mpas_ocn_init_routines.F

```diff
@@ -459,6 +460,37 @@ subroutine ocn_init_routines_vert_coord(domain)!{{{
 
             vertCoordMovementWeights = 1.0_RKIND
 
+         ! Set weight tapering:
+         !  1.0 shallower than depth1
+         !  linear in between
+         !  0.0 deeper than depth2
+         elseif (config_vert_coord_movement.eq.'tapered_250m') then
```

Comment:
Should we make these case constructs?

Reply:
I find a case statement harder to read.  I personally prefer if/elseif.

Reply:
I think if/elseif is more readable too. This is in init, so not a performance issue.

Reply:
Phil Jones' seminar last week suggested it was better from a performance standpoint, hence my recommendation.  Its irrelevant now anyway.

Reply:
Any self-respecting compiler should be able to see that and optimize it on its own.  I think it's a fool's errand to worry about such things unless they're deep in a loop or something.